src/html.ts

```diff
@@ -25,20 +26,36 @@ navigator.serviceWorker.register('${path}', { scope: '${options.scope}' })
 }`.replace(/\n/g, '')
 }
 
+export function checkForHtmlHead(html: string) {
+  if (!html.includes('</head>')) {
```

Comment:
should we also check for `<head />`? This will require to parse the html: Vite doesn't add the entries if `head` is missing and returning the tag array instead here